### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Timi2424**
https://monosnap.com/file/d0OBFibL4Hi4tjJv79PjpkOsYzFUAG
- [DEMO LINK](https://timi2424.github.io/layout_stars/src/)
- [TEST REPORT LINK](https://Timi2424.github.io/layout_stars/backstop_data/html_report/)
I had to reroute the link, because I couldn't deploy it. So instead I deployed the page via branch develop and reroute links to proper directory.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  margin-right: 4px;
+  width: var(--star-size);
+  height: var(--star-size);
+}
+
+.stars__star:last-child {
+  margin: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
```
Comment (mbruhler) [inline src/style.css:33]: Combine same block under multiple selectors.

You can do:
```
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2),
... other selectors
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+.stars {
+  display: flex;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.stars__star {
+  background: url(./images/star.svg) no-repeat center;
+  display: inline-block;
+  margin: 0;
```
Comment (mbruhler) [inline src/style.css:23]: Don't use `margin: 0` when you set one line below `margin-right: 4px;`

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
+:root {
+  --star-size: 16px;
+}
+
+* {
+  margin: 0;
+}
+
+.header {
+  width: 96px;
```
Comment (mbruhler) [inline src/style.css:10]: Also it's a good idea to add it to css vars

**Review by mbruhler — CHANGES_REQUESTED** [review]
Your links still don't work also in this PR -> https://%3Ctimi2424%3E.github.io/layout_stars/
Please fix it, refer to documentation in readme file to fix it.

Also there are couple of issues in your styling.

Good luck!

File: src/index.html [hunk 4]
```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <header class="header">
+      <div class="stars stars--0">
+        <p class="stars__star"></p>
```
Comment (mvjl000) [inline src/index.html:13]: Why are you using here `p` tags?

Comment (mvjl000) [reply src/index.html:13]: I would replace them with divs

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+.stars {
+  display: flex;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.stars__star {
+  background: url(./images/star.svg) no-repeat center;
+  display: inline-block;
+  margin: 0 4px 0 0;
```
Comment (mvjl000) [inline src/style.css:24]:
```suggestion
  margin-right: 4px;
```

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+.stars {
+  display: flex;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.stars__star {
+  background: url(./images/star.svg) no-repeat center;
+  display: inline-block;
```
Comment (mvjl000) [inline src/style.css:23]: I think you can leave default here.

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+.stars__star:last-child {
+  margin: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background: url(./images/star-active.svg) no-repeat center;
```
Comment (mvjl000) [inline src/style.css:38]: Actually you only want to change background-image
```suggestion
  background-image: url(./images/star-active.svg);
```

**Review by mvjl000 — CHANGES_REQUESTED** [review]

**Review by mbruhler — CHANGES_REQUESTED** [review]
Please fix test report link (404)

Also add changes from @mvjl000 and we will be good to go.

**Pull request closed without merging**